model/channel.go

```diff
@@ -24,7 +24,8 @@ type Channel struct {
 	Type          string `json:"type"`
 	DisplayName   string `json:"display_name"`
 	Name          string `json:"name"`
-	Description   string `json:"description"`
+	Header        string `json:"header"`
+	Purpose       string `json:"purpose"`
```

Comment:
I don't think this will work on postgres upgrade.  New columns must be appended.